.github/workflows/last-opp-statiske-filer-til-gcs-dev.yaml

```
@@ -34,6 +34,15 @@ jobs:
               run: npm run build:dev
               env:
                   PUBLIC_URL: https://veilarbdetaljer.intern.dev.nav.no
+            - name: Upload to CDN
+              uses: nais/deploy/actions/cdn-upload/v2@master
+              with:
+                team: obo
+                source: ./build
+                destination: dev/overblikk-modia
```

Syns vi kan bruke samme konvensjon som Dab har gjort, med appnavnet, så blir det lettere å skjønne hvilket repo filene kommer fra!
```suggestion
                destination: veilarbdetaljerfs-dev
```